### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by R-Ramana**
Addresses #42 

Updated the following in the user guide:

- Table of Contents (for quick access)
- Introduction
- Quick Start
- Features List
- FAQ
- Command Summary

Following changes in the dev guide:

- Table of Contents (standardize sizing across documents)

Changes in the docs/README:

- NotUS product intro

Requires further updates but I think it is good to go for a starter!

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,42 +1,402 @@
...
+
+Expected output:
+
+
+### <a id="delete-n"><ins>3.8 Delete note:</ins> `delete-n`</a>
+Deletes an existing note.
+
+Format: `delete-n [/i INDEX] [t/TITLE]`
+
+- Pins a note to the top of the list. A pinned note will be unpinned. The index refers to the index number shown in the displayed note list. The index must be a **positive integer** (1, 2, 3, …).
```
Comment (Chongjx) [inline docs/UserGuide.md:206]: The message is from pin command

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,42 +1,402 @@
...
+
+💡 Creating a tag that already exists will override the tag color.
+
+Example of usage: 
+
+`create-t /tag CS2113`
+
+`create-t /tag important red /tag CS2113 blue`
+
+💡 If no tag color is specified, the color of the tag will be that of the default console color.
```
Comment (Chongjx) [inline docs/UserGuide.md:235]:
```suggestion
💡 If no tag color is specified, the color of the tag will be that of the default white color.
```

**Review by Chongjx — APPROVED** [review]
Looks good overall, just some minor typo

**Review by prachi2023 — APPROVED** [review]
Loooks good. Love the Introduction NOTUS logo

**Pull request merged**